adapters/resetdigital/resetdigitaltest/examplary/simple-banner.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
`adapters/resetdigital/resetdigitaltest/examplary/simple-banner.json` is not being referred by unit tests added in `resetdigital_test.go`


Additionally, should add supplemental tests as well. Refer appnexus adapter as example - https://github.com/prebid/prebid-server/blob/dd07431ef54c38a2617c04e76b3e7bda9c21f569/adapters/appnexus